lib/vanagon/component/dsl.rb

```diff
@@ -281,6 +281,16 @@ def link(source, target)
         @component.add_file Vanagon::Common::Pathname.file(target)
       end
 
+      # Add a %ghost entry to the rpm %files section.
+      # This indicates a file that is tracked in the rpm database (the package
+      # manages it), but not installed. Used, for example, when setting up
+      # alternative packages for use with update-alternatives.
+      #
+      # @param file [String] installed path of the file.
```

Comment:
you've called the param `file` here, but `path` below

Reply:
Oops, thanks; I'll switch that to path in the docstring.

Reply:
fixed